src/utility.jl

```diff
@@ -74,3 +74,115 @@ function visualize_flow(method::ColorBased, flow::Array{SVector{2, Float64}, 2};
     return hsv
     
 end
+
+struct Point{T}
+    x::T
+    y::T
+end
+
+function check_flow(p::Point)
```

Comment:
perhaps this should be `is_flow_unknown`, or `is_flow_known`? I'm guessing you are referring to the `.flo` specification 

> // A flow value is considered "unknown" if either |u| or |v| is greater than 1e9.

If we are working with `p::SVector{2,Float64}` then you can access the first and second element of `p` using
`first(p)` and `last(p)`. 

```julia
return !isnan(first(p)) && !isnan(last(p)) && abs(first(p)) < 1e9 && abs(last(p)) < 1e9
````

Alternatively, you could use broadcasting as well, e.g.
```julia
all(abs.(p) .< 1e9)
```